src/lib/styles/tools/form-fields/_inline-field-layout.scss

```diff
@@ -62,12 +70,45 @@
 
     @include breakpoint.up(forms.$horizontal-breakpoint) {
         &.rootLayoutHorizontal {
+            grid: inherit;
+            grid-template-columns: subgrid;
+            grid-column: span 2;
+
+            @supports not (grid-template-columns: subgrid) {
+                display: contents;
+            }
+        }
+
+        &.rootLayoutHorizontal .field {
+            display: contents;
```

Comment:
Just to be sure. I remember that the last time there was an issue with `display: contents;`, we met it few days/week ago. I just want to be sure it is fine in this contexts.

Reply:
Good point 🙂. This is OK because, as opposed to the case with the cart table, we don't inherit any styling from the `.field` class. The only thing it does is defining the layout which is exactly what we need to destroy by `display: contents`.